pkg/volume/volume.go

```diff
@@ -171,6 +177,29 @@ type Detacher interface {
 	UnmountDevice(deviceMountPath string) error
 }
 
+// NewTryAgainError returns a new instance of TryAgainError error.
+func NewTryAgainError(message string) error {
```

Comment:
To be more concrete, I would have the various plugins return a common `DeletedVolumeInUse` error in this case. And then have the controller implement retry in response to that error.